.github/actions/spelling/excludes.txt

```diff
@@ -2,63 +2,82 @@
 (?:^|/)(?i)COPYRIGHT
 (?:^|/)(?i)LICEN[CS]E
 (?:^|/)3rdparty/
+(?:^|/)\.keep$
```

Comment:
This was automatically suggested by check-spelling -- it refers to a file named `.keep`